chemprop/args.py

```diff
@@ -546,14 +582,37 @@ def process_args(self) -> None:
                 for key, value in config.items():
                     setattr(self, key, value)
 
+        # Determine the target_columns when training atomic and bond targets
+        self.is_atom_bond_targets = False
+        if self.atom_targets != [] or self.bond_targets != []:
+            self.is_atom_bond_targets = True
+            self.target_columns = self.atom_targets + self.bond_targets
```

Comment:
Does this just overwrite the `--target_columns` argument? That was useful for allowing certain targets in a data file to be ignored. That would sill probably be useful for atom/bond operations.

Reply:
Because the current version cannot train atom-, bond-, and molecule-level targets at the same time, it won't overwrite the `--target_columns` when training on atom- and bond-level targets. However, if training on atom-, bond-, and molecule-level targets is expected, this code should be modified.

Reply:
I have modified this part. The `--atom-targets` and `--bond-targets` arguments have been removed. User can use `--target_columns` to specify the targets they want to train. However, if atomic or bond targets have been specified, the molecule-level targets would be ignored.